build.sc

```diff
@@ -15,18 +15,17 @@ import mill.scalanativelib.api.{LTO, ReleaseMode}
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 import com.github.lolgab.mill.mima._
 
-val scala211  = "2.11.12"
 val scala212  = "2.12.17"
 val scala213  = "2.13.10"
-val scala3   = "3.1.3"
-val scalaJS  = "1.10.1"
-val scalaNative = "0.4.5"
+val scala3   = "3.2.2"
```

Comment:
Yeah, good point! Reverting it.

Reply:
Sorry for the delay. I think the reason here, was that I was attempting to get some of the extra ujson integrations working. 

I think with scala at 3.1.3, this command will fail, because circe has been rolled forward to scala 3.2.2. 
`mill ujson.circe[3.1.3].compile`

```
[error] class file io/circe/package.class is broken, reading aborted with class dotty.tools.tasty.UnpickleException
[error] TASTy signature has wrong version.
[error]  expected: {majorVersion: 28, minorVersion: 1}
[error]  found   : {majorVersion: 28, minorVersion: 2}
[error]
```


So I guess, that in current state upickle this would fail to publish correctly. However, I think a better workaround may be to use circe 0.14.1. That at least appears to compile, and I would guess, is the preferred compromise - it is not rational for circe to drive the scala version upickle supports! 

I'll put up the teeny tiniest of PRs.

Reply:
We might need to bump Scala 3 for other reasons, see https://github.com/com-lihaoyi/upickle/pull/454
So let's wait for the other PRs to land before changing anything.